data/hai/hai reveal 5 philosophers.txt

```diff
@@ -212,22 +212,72 @@ mission "Hai Reveal [B03] Contact Giti"
+				goto briefing
+
+			label offended
+			`	Everyone looks surprised or offended, as if you are the court jester who thought he was king. They're dismissive of your comment, as they begin to discuss your departure.`
+				goto done
+			label honored
+			`	The leaders all look pleased. General Choot'k says, "You've surpassed my expectations, Captain <last>. I've never seen a fleet so large, but you led it to victory with the speed and skill of a veteran general." Danforth looks just as impressed.`
+			label done
+			action
+			`	Danforth explains, "For now, all communication in and out of the area is blocked at the relay unless its to or from specific high-level Navy or Syndicate sources. Captain <last>, I'm afraid that also includes you. We don't have time or equipment to harden your systems against potential intrusion, but you do have the nearly exclusive freedom to pass through the blockade. You're a pilot, not a ground combat troop operator, so you should pursue other matters."`
 			`	On your way out Sayari catches you and suggests meeting aboard Teeneep's ship to discuss some things.`
-			`	Once there she pulls up a range of information on known or possible smuggling operations that they, the Republic, and Syndicate had reconciled for justifying this move on <origin>. Her focus though, is on what they hadn't reconciled. "So, unfortunately, while many matches were found - which led to this occupation of Stormhold - there was nothing in the new data sets which was able to be conclusively linked to anything associated with the mysterious symbol. And no human escapees had ever been tested for mild cadmium poisoning, so that was a dead end too. I know it's been thrust into the background pretty heavily, but did you find anything yourself? We need to decide if we're setting this aside or not."`
+			`	Once there she pulls up information on known or possible smuggling operations used to justify the <origin> operation. Her focus is on what they hadn't reconciled. "Unfortunately, while many matches were found there was nothing in the new data that conclusively linked to the mysterious symbol. No human escapees had been tested for mild cadmium poisoning, so that was a dead end too. I know it's been thrust into the background pretty heavily, but did you find anything yourself? We need to decide if we're setting this aside or not."`
```

Comment:
```suggestion
			`	Once there she pulls up information on known or possible smuggling operations used to justify the <origin> operation. Her focus is on what they hadn't reconciled. "Unfortunately, while many matches were found, there was nothing in the new data that conclusively linked to the mysterious symbol. No human escapees had been tested for mild cadmium poisoning, so that was a dead end too. I know it's been thrust into the background pretty heavily, but did you find anything yourself? We need to decide if we're setting this aside or not."`
```